### ISSUE TITLE
`-[SLElement isVisible]` reports elements in non-key windows as not visible.

### ISSUE THREAD

**Issue by wearhere** [inkling/Subliminal#212]
For instance, picker views that are input views: https://github.com/inkling/Subliminal/pull/211/files#diff-c961c449e151dca76856c2af83d814a6R33.

The first problem is that `-[UIView slAccessibilityIsVisible]` only renders the key window: https://github.com/inkling/Subliminal/blob/master/Sources/Classes/Internal/NSObject+SLVisibility.m#L279. That line should be changed to `UIWindow *window = self.window;`.

However, that change requires that we then consider the visibility of the window itself: even if an element is visible within its window, it should not be reported as visible if the window is e.g. covered by another window.

**Comment by wearhere** [comment]
This should wait until #211 goes in because until then, there's no way to match elements in other windows.
